build.gradle

```diff
@@ -629,6 +634,14 @@ task smoke(type: Test) {
         includeTags 'smoke'
     }
 
+    doFirst {
```

Comment:
some explanatory comments here would be good too. I know we don't have many comments in the build.gradle but we should start getting in the habit to add them other wise it's difficult to understand why a piece of config is there